.pipelines/dwsql-pipelines.yml

```diff
@@ -50,27 +52,43 @@ jobs:
       dockerVersion: 17.09.0-ce
       releaseType: stable
 
+  - task: Bash@3
+    displayName: 'Generate password'
+    inputs:
+      targetType: 'inline'
+      script: |
+        password=$(cat /dev/urandom | tr -dc 'A-Za-z0-9' | head -c24)
```

Comment:
Using special characters leads to the following error in the `Get and Start Ubuntu SQL Server Image Docker with SSL enabled`

Error: `Detected characters in arguments that may not be executed correctly by the shell. Please escape special characters using backslash (\)`

Reply:
Is this all special characters? I only saw documentation that complained about `@` and `$`  (this is because semi colon `;` and spaces ` ` and quotes `'` `"` were not in the regex here.)
https://learn.microsoft.com/en-us/sql/relational-databases/security/strong-passwords?view=sql-server-ver16#:~:text=Starts%20with%20the%20%24%20or%20%40%20character.